### PR TITLE
Fire direction fix

### PR DESCRIPTION
**Pull request by frozenbanana**
## Purpose
  - make projectile always pass trough crosshair

## Changes
 - Calculation of fire direction in `WeaponSystem.cpp`
 - Offsets to make weapon not floating in air. (might be replaced later with hands)

## Testing
- [x] Tested in Sandbox

### CODE REVIEW

File: CrazyCanvas/Source/ECS/Systems/Player/WeaponSystem.cpp [hunk 1]
```diff
@@ -294,7 +300,7 @@ glm::vec3 WeaponSystem::CalculateZeroingDirection(
 	glm::vec3 zeroPoint = glm::vec3{ playerPos.x, weaponPos.y, playerPos.z } + glm::normalize(GetForward(playerDirection)) * zeroingDistance;
 	glm::vec3 fireDirection = glm::normalize(zeroPoint - weaponPos);
 
-	glm::quat directionQuat = glm::identity<glm::quat>();
-	SetForward(directionQuat, fireDirection);
-	return glm::rotate(GetForward(glm::normalize(directionQuat)), glm::radians(m_YAngle), GetRight(glm::normalize(directionQuat)));
+	// adjusts m_YAngle to be less inpactful when aiming up or down.
```
Comment (TheoBerlin) [inline CrazyCanvas/Source/ECS/Systems/Player/WeaponSystem.cpp]:
```suggestion
	// adjusts m_YAngle to be less impactful when aiming up or down.
```

File: CrazyCanvas/engine_config_crazycanvas.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "CONFIG_OPTION_MESH_SHADER": false,
     "CONFIG_OPTION_SHOW_RENDER_GRAPH": false,
     "CONFIG_OPTION_RENDER_GRAPH_NAME": "DEFERRED_PBR.lrg",
-    "CONFIG_OPTION_LINE_RENDERER": false,
+    "CONFIG_OPTION_LINE_RENDERER": true,
```
Comment (TheoBerlin) [inline CrazyCanvas/engine_config_crazycanvas.json]: no no

**Review by TheoBerlin — CHANGES_REQUESTED** [review]
The line renderer should not be enabled

File: CrazyCanvas/engine_config_crazycanvas.json (hunk 2 above)
Comment (frozenbanana) [reply CrazyCanvas/engine_config_crazycanvas.json]: good catch. I have bytat now.

**Review by TheoBerlin — APPROVED** [review]

**Pull request merged**